pfb-analysis/scripts/setup_database.sh

```diff
@@ -7,15 +7,32 @@ NB_POSTGRESQL_USER=gis
 NB_POSTGRESQL_PASSWORD=gis
 
 # Set defaults for overridable configuration params
-PFB_WORK_MEM="${PFB_WORK_MEM:-2048MB}"
 PFB_CHECKPOINT_COMPLETION="${PFB_CHECKPOINT_COMPLETION:-0.8}"
-PFB_MAX_WAL_SIZE="${PFB_MAX_WAL_SIZE:-2GB}"
+# Disable to improve performance, if disabled, data loss occurs on server crash (ok for analysis)
+PFB_FSYNC="${PFB_FSYNC:-off}"
```

Comment:
Currently testing a run with these options merged with our existing ones.

I left our existing `max_wal_size` setting at 256MB. The formula suggested by the 9.5 changelog is `max_wal_size = (3 * checkpoint_segments) * 16MB` which would be ~12GB if we used the `checkpoint_segments` value above, which we don't have the disk space for (nor do I think such a large value would be particularly helpful). Hopefully its not the case that leaving `max_wal_size` unchanged adversely impacts the other changes.